src/test/regress/expected/alter_table.out

```diff
@@ -4186,7 +4188,7 @@ DROP TABLE fail_part;
 -- fails with incorrect object type
 CREATE VIEW at_v1 AS SELECT 1 as a;
 ALTER TABLE at_v1 ATTACH PARTITION dummy default;
-ERROR:  "at_v1" is not a partitioned table or index
+ERROR:  "at_v1" is not a partitioned table or index (parse_utilcmd.c:5075)
```

Comment:
Can you use initfile to filter out `parse_utilcmd.c:5075`? Otherwise, each time modification of the file, might lead to the case failure.

Reply:
Good idea.

Reply:
Fixed in this commit: ec034aef286ef14a4d43e773d318bcac6e8626b3